.dockerignore

```diff
@@ -0,0 +1,46 @@
+.git/
+*~
+*.bak
+__pycache__
+opentitan-docs
+
+# fusesoc result
+build/
+
+# build directories with special suffixes
+build*/
```

Comment:
This catches `build/` doesn't it?

Reply:
Oh.. yes. I will delete above Fusesoc ignore list :)